eng/azure-pipelines.yml

```diff
@@ -84,6 +86,21 @@ stages:
           ${{ if eq(variables.officialBuild, 'true') }}:
             displayName: Build and publish illink.sln $(_BuildConfig)
 
+      - job: SourceBuild_Managed
```

Comment:
I think this should disable tests processing and publishing if they are not built

Reply:
Sorry, I am lost. Can you please help me understand how to do that?

Reply:
Right now, the flag comes in through here:

https://github.com/mono/linker/blob/d1bb874351fd4652edd83285f4488739ea9fda7d/eng/azure-pipelines.yml#L36-L43

and get passed thorough the jobs template into the job template here:

https://github.com/mono/linker/blob/d1bb874351fd4652edd83285f4488739ea9fda7d/eng/common/templates/jobs/jobs.yml#L40-L52

*Maybe* you can add a `enablePublishTestResults: false` to the source-build job and it would override the "outer scope" `enablePublishTestResults: true` parameter. But AzDO might just throw an error if it's defined twice.

A more reliable solution might be to completely remove the "outer scope" `enablePublishBuildArtifacts: true`, and put it on *each* job that *should* publish test results.